Sources/PrivacyDashboard/PrivacyDashboardController.swift

```diff
@@ -368,10 +368,13 @@ extension PrivacyDashboardController: PrivacyDashboardUserScriptDelegate {
     }
 
     func userScriptDidRequestShowReportBrokenSite(_ userScript: PrivacyDashboardUserScript) {
```

Comment:
This is a legacy message that will isn’t currently being fired and is likely to be removed in the future.